src/lunariumcmd.src

```diff
@@ -1479,13 +1482,15 @@ cmds.escalate=function(sin)
     out.bool=2
     globals.args=null
 
-	if shelltype != "file" then
-		if not comp.is_network_active() then
-			return error("Internet connection required")
-		end if
-	end if
+	//if shelltype != "file" then // this is escalation, i don't entirely believe it requires internet connection.
+	//	if not comp.is_network_active() then
+	//		return error("Internet connection required")
+	//	end if
+	//end if
 
 	// as escalate does not works on a remote connection - why allowing user to use it on it?
+	// because i decided it would be nice to still be able to escalate/hack on the local machine while connected to something else remotely. it was actually rather useful! maybe instead a warning that its escalating the local machine would've been better though -Clover
```

Comment:
Good point, but then it should actually tell the user it does the escalation at the machine which runs the Lunarium(Or Lunar).

Reply:
indeed, at the end i did suggest i could have included a warning that it was escalating on the machine that ran lunar